_posts/2022-04-01-meeting-minutes.md

```diff
@@ -0,0 +1,101 @@
+# What We're Going Over:
+- Ownership transfers
+- ACM Picnic
+- Non-computer science students outreach 
+- SIG Functional
+- Trello Board
+- Website Domain Renewal 
+
+# What We Went Over:
+- Bank Account
+    - 901.91
+    - Amanda and Carson going on the bank account
+    - Sean and Ryan going off the bank account
+- Ownerships are transfered
+- Still need to do financial transfer
+- ACM Picnic
+- Last year it was the week before dead week
+- It will be in Island Park again
+- “Get better hotdogs this time” -Sean
+- Badminton, Volleyball, and Nerf Guns
+- We need a net
+- Can we rent from the student union?
+- Get quickstart charcoal 
+- New Member at Large
+- Walmart discount
+- Will check on Sam’s Club
+- Bytle
+- Need a start date
+- Who is interested:
+    - Board members
+    - Sean
+        - Braxton
+    - Nick
+    - Gabe
+- End of year stuff to look ahead to
+- Debit board (Carson)
+- Restart monthly emails (John)
+- Fall semester weekly meeting topics
+- Make sure to redo Git and Data Mining
+- Make a google form for “guest”(members) speakers
+- Advertise form at kick off meeting
+- Member at Large brings member ideas to board meetings
+- We would need to define standards for SIGs
+- SIG Roles on discord for quick announcements
+- Plan date for Fall Kickoff Meeting
+- SIGS
+- GDev (Alan)
+- Need to advertise more
+- Functional Programming (Brandon)
+- Compete 
+- Committee members for running SIGs who report to Vice Chair
+- Welcome Week (Amanda + Jean)
+- Want two people to present during welcome week
+- Voting on making a Committee for SIGs (Passed unanimous)
+- Committee member must present a SIG document to the Board
+- Committee members are elected by a majority vote by the Board
+- Discord bot for auto-giving roles to acm members who want SIG notifications
+- Committee members then get a role allowing them to @ the SIG notification roles
+- Voting on when to have a SIG explanation meeting
+- Before or after the kick off
+- 3 Afters, 1 Before, 1 Abstin
```

Comment:
I'm not sure if we need to document all the votes? I'd have to read up the constitution and ask old board members